server/src/main/scala/io/delta/standalone/internal/DeltaSharedTableLoader.scala

```diff
@@ -159,12 +159,15 @@ class DeltaSharedTable(
           } else {
             selectedFiles
           }
-        val sampleFiles =
-          if (sampleHint.nonEmpty) {
-            Sampler.sample(filteredFiles, sampleHint.get)
-          } else {
-            filteredFiles
-          }
+        val (precisionFrom, precisionTo) = sampleHint match {
```

Comment:
Here the precision defined in the configuration is combined with the precision range specified in the request. If there is no sample hint in the request, then only predefined precision is used.

Reply:
Perfect, and it takes the min of them. Cool, so this should work also when we build a "Qbeast aware" client.